spec/appendix.md

```diff
@@ -66,6 +69,9 @@ See also: decentralized identifier, [ref: self-certifying identifier (SCID)].
 [[def: key event log (KEL), KEL, KELs]]
 ~ A verifiable append-only log of [[ref: key events]] for an [[ref: AID]] that is both backward and forward-chained, as defined by the [[ref: KERI specification]]. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/key-event-log) for more detail.
 
+[[def: KEL backed data]]
+~ [[ref: KEL]] backed data in `did:webs` provides the highest level of secure data and can either be found in the KEL or found anchored to the KEL. This means the signatures on the events in the KEL are strongly bound to the key state at the time the events are entered in the KEL. This provides the strongest guarantee of duplicity evidence so that any verifier is protected. The information is end-verifiable and any evidence of duplicity means do-not-trust. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/kel-backed-data) for more detail.
```

Comment:
```suggestion
~ [[ref: KEL]] backed data in `did:webs` provides the highest level of data security assurance and such data can be found either in the KEL or anchored to an event in the KEL. This means that the signatures on the events in the KEL are strongly bound to the key state at the time the events are entered in the KEL, that is the data. This provides strong guarantees of non-duplicity to any verifiers receiving a presentation as the KELs are protected and can be watched by agents ([[ref: watcher]]) of the verifiers. The information is end-verifiable and any evidence of duplicity in the events is evidence that the data or presentation should not be trusted. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/kel-backed-data) for more detail.
```

Reply:
This def sounded a little strange so I reworded it a bit.  Feel free to take some or all of it (or none of it if you think the original was good).